docs/sections/header.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/main.js"></script>
+	</head>
+	<body>
+        <header class="navbar navbar-expand-lg p-0 flex-lg-wrap">
+            <div class="announcement-bar bg-primary-light text-center w-100">
+                <div class="container justify-content-center font-weight-bold">
+                    <a href="#">Up to 25% off + Free Gift worth $25.40</a>
+                </div>
+            </div>
+            <nav class="container bg-white">
+                <button class="navbar-toggler text-left col-1 px-0" type="button" data-trigger="#navbar_main">
+                    <span class="navbar-toggler-icon d-block"></span>
+                </button>
+
+                <ul class="navbar-collapse collapse w-100 px-0 navbar-nav align-items-center">
+                    <li class="nav-item">
+                        <a class="nav-link dropdown-toggle text-uppercase font-weight-bold text-white btn btn-primary mr-1" href="#" aria-haspopup="true" aria-expanded="false" data-target="megaMenuDropdown">Shop</a>
```

Comment:
@foria I set .mega-menu__button for the li element, because there is an issue on mega menu below when on the a element

https://gifyu.com/image/YwsG